shell/browser/api/electron_api_session.h

```diff
@@ -150,6 +174,12 @@ class Session : public gin::Wrappable<Session>,
   void OnExtensionUnloaded(content::BrowserContext* browser_context,
                            const extensions::Extension* extension,
                            extensions::UnloadedExtensionReason reason) override;
+
+  void SetExtensionAPIHandlers(const gin_helper::Dictionary& api,
+                               gin::Arguments* args);
+  std::unique_ptr<ExtensionTabDetails> GetExtensionTabDetails(
```

Comment:
For returning nullable value, the recommended coding style is to return `base::Optional<ExtensionTabDetails>`.